CHANGES_NEXT_RELEASE

```diff
@@ -1,2 +1,4 @@
 - Fix: Correct rendering of response to convop "POST /v1/contextEntities/{entityId::id}/attributes/{attributeName} (Issue #772)
-- Fix:  All convenience operations to use the service routines of standard operations (Issue #117)
+- Fix: All convenience operations to use the service routines of standard operations (Issue #117)
+- Fix: Made 'POST /v1/contextEntities/{entityId::id}' and 'POST /v1/contextEntities' accept entity::id and entity::type in payload,
```

Comment:
I found a sentence that is 100% true (Remember: entity in payload was forbidden for '/v1/contextEntities/EEE', while /v1/contextEntities didn't have any URI params):

- Fix: Made 'POST /v1/contextEntities/{entityId::id}' and 'POST /v1/contextEntities' accept entity::id and entity::type
       in both payload and URL, as long as the values coincide.